gnucash/python/init.py

```diff
@@ -6,6 +6,10 @@
 require_version('Gtk', '3.0')
 from gi.repository import Gtk
 import os
+import gettext
+# install gettext _-function with path to mo files
+gettext.install("gnucash",
+        os.path.join(os.environ["GNC_HOME"],"share/locale"))
```

Comment:
GNC_HOME is not available for the python bindings. So this will work only for the python shell. In addition I am not sure if GNC_HOME/share/locale will always be the right place.

Reply:
There is a function gnc_path_get_localedir. I changed _sw_core_utils to make this accessible.

Reply:
@fellen I tried to follow the way c and scheme use the gettext API. I didn't go the whole way though. As far as I understand the translation in python takes place in runtime (or at least possibly) so that the mo files in the target system are needed.

I hope that the question of the right path has been solved by https://github.com/Gnucash/gnucash/pull/506/commits/859808ecd7df4cb0a72a179daa9a4e2634f4a2dd.

I don't think that I overwrite something as the python gettext package just doesn't translate without setting a path. It does the same thing as c gettext but in a seperate way and needs a separate initialization. I don't think that it's possible in a simple way to access the exact same way c and scheme do by swig wrapping  those. But I may be wrong.

Reply:
Python's gettext module is just a wrapper around the gettext library.  Try this in pycons:
```python
>>> from gettext import textdomain
>>> textdomain()
```
If it says 'gnucash' there's no need to set it for pycons. Scripts using the bindings will of course have to set up gettext on their own.

Reply:
textdomain() outputs "messages", not "gnucash". In pycons and in the python bindings.

Reply:
I have started https://wiki.gnucash.org/wiki/I18N#Using_Other_Programming_Languages for this project. See the Note there and https://github.com/Gnucash/gnucash/blob/maint/gnucash/gnucash-bin.c starting from the end.

Reply:
Looks like I'm wrong about python's [gettext module](https://github.com/python/cpython/blob/3.7/Lib/gettext.py). It's a reimplementation written mostly in python, so there's no way for it to share textdomain setting with the C library.